core/src/components/refresher-content/refresher-content.tsx

```diff
@@ -108,7 +108,7 @@ export class RefresherContent implements ComponentInterface {
           {this.pullingIcon && hasSpinner && (
             <div class="refresher-pulling-icon">
               <div class="spinner-arrow-container">
-                <ion-spinner name={this.pullingIcon as SpinnerTypes} paused></ion-spinner>
+                <ion-spinner name={this.pullingIcon as SpinnerTypes} aria-label="pulling icon" paused></ion-spinner>
```

Comment:
`ion-refresher-content` is [failing](https://dequeuniversity.com/rules/axe/4.8/aria-progressbar-name?application=playwright) to pass Axe testing due to not passing an `aria-label` to `ion-spinner`.